config/routes.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  resources :messages
```

Comment:
- [ ] OPTIONAL:  Resourses keyword creates many endpoints, but for this project you need only one, index, so I strongly recommend to have one route: 
`resources :messages, only: [:index]`
Try to keep projects clean and it will help you to have good coding habits 👍🏻